### PR TITLE
CodeBuild Integration update

### PR DESCRIPTION
**Pull request by jchrisfarris**
Adds the integration for CodeBuild that was done at Integrate, and fixes the usage of a deprecated envvar

### CODE REVIEW

**Comment by jchrisfarris** [comment]
I'm gonna put the full buildspec in samples and add a link. So lets hold on merging for the moment

**Comment by jchrisfarris** [comment]
Please merge https://github.com/turbot/steampipe-samples/pull/6 before merging this PR.

File: docs/integrations/aws_codebuild.md [hunk 1]
```diff
@@ -42,15 +42,40 @@ Running streampipe in CodeBuild uses the same `run-as: codebuild-user` as the in
...
 Because the `steampipe check all` command returns the number of violations and we want CodeBuild to exit cleanly, we add `on-failure: CONTINUE` to the build phase.
 
-For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_WORKSPACE_CHDIR`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_MOD_LOCATION`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
 
 
-That's it! Now you can use any of Steampipe's plugins and mods as part of your CodeBuild projects.
+## Using Steampipe Cloud
+
+Steampipe can also push snapshots to Steampipe Cloud via AWS CodeBuild. To do this we make a few changest to the buildspec file.
```
[the remark this answers is not in the document]
Comment (jchrisfarris) [reply docs/integrations/aws_codebuild.md]: Fixed in original branch along with a reference to `streampipe`

File: docs/integrations/aws_codebuild.md [hunk 2]
```diff
@@ -42,15 +40,43 @@ Running streampipe in CodeBuild uses the same `run-as: codebuild-user` as the in
...
 
 Because the `steampipe check all` command returns the number of violations and we want CodeBuild to exit cleanly, we add `on-failure: CONTINUE` to the build phase.
 
-For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_WORKSPACE_CHDIR`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_MOD_LOCATION`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+
+
+## Using Steampipe Cloud
+
+Steampipe can also push snapshots to Steampipe Cloud via AWS CodeBuild. To do this we make a few changes to the buildspec file.
```
Comment (johnsmyth) [inline docs/integrations/aws_codebuild.md:55]: Maybe link to https://steampipe.io/docs/cloud/overview the first time we mention Steampipe Cloud?  I would maybe add a link for 'push snapshots' to either https://steampipe.io/docs/snapshots/overview or https://steampipe.io/docs/snapshots/batch-snapshots to provide a little more context

File: docs/integrations/aws_codebuild.md [hunk 3]
```diff
@@ -42,15 +40,43 @@ Running streampipe in CodeBuild uses the same `run-as: codebuild-user` as the in
...
+  secrets-manager:
+    STEAMPIPE_CLOUD_TOKEN: steampipe-cloud:STEAMPIPE_CLOUD_TOKEN
+    WORKSPACE: steampipe-cloud:WORKSPACE
+```
+
+Next, replace the last line of the build with a call to Steampipe Cloud:
+```yaml
+      - cd terraform ; ../steampipe check all --snapshot-location $WORKSPACE --snapshot --snapshot-title "Terraform Report"
+```
+This command will run steampipe and save the output of the check as "Terraform Report" in the specified Workspace. By default, the CLI looks for your steampipe cloud token in the `STEAMPIPE_CLOUD_TOKEN` [environment variable](https://steampipe.io/docs/reference/env-vars/overview).
```
Comment (johnsmyth) [inline docs/integrations/aws_codebuild.md:70]: I think we should add a little context for how to generate steampipe cloud token.  You probably want to [create a non-expiring token](https://steampipe.io/docs/cloud/profile#tokens), but could alternately run [steampipe login](https://steampipe.io/docs/reference/cli/login) and copy the temporary token (30 day expiration currently)

**Review by johnsmyth — CHANGES_REQUESTED** [review]

**Review by johnsmyth — APPROVED** [review]

**Pull request merged**